rancher-fleet.yaml

```diff
@@ -35,6 +36,59 @@ subpackages:
         - runs: |
             fleetagent --version | grep ${{package.version}}
 
+  - name: ${{package.name}}-controller
+    pipeline:
+      - uses: go/build
+        with:
+          ldflags: |
+            -X github.com/rancher/fleet/pkg/version.GitCommit=$(git rev-parse HEAD)
+            -X github.com/rancher/fleet/pkg/version.Version=${{package.version}}
+          output: fleetcontroller
+          packages: ./cmd/fleetcontroller
+    test:
+      environment:
+        contents:
+          packages:
+            - curl
+      pipeline:
+        - runs: |
+            fleetcontroller --version
+            fleetcontroller --help
+        - name: "Setup KWOK cluster"
+          uses: test/kwok/cluster
+        - name: "Install CRDs"
+          runs: |
+            kubectl apply -f https://raw.githubusercontent.com/rancher/fleet/refs/tags/v${{package.version}}/charts/fleet-crd/templates/crds.yaml
+            kubectl wait --for=condition=Established crd --all --timeout=60s
+            kubectl create ns cattle-fleet-system
+        - name: "Start fleet controller daemon"
+          uses: test/daemon-check-output
+          with:
+            start: |
+              fleetcontroller \
+                --namespace cattle-fleet-system  \
+                --kubeconfig ~/.kube/config
+            timeout: 30
+            expected_output: |
+              listening for changes on local cluster
+              starting job scheduler
+              starting manager
+              successfully acquired lease
+            post: |
+              # Check metrics endpoint
+              curl -sk http://localhost:8080/metrics
+
+  - name: ${{package.name}}-termination-log
+    description: fleet-termination-log outputs fleet's termination log to /dev/termination-log
+    dependencies:
+      runtime:
+        - busybox
```

Comment:
let's use full coreutils in here.

Reply:
Fleet uses [busybox as base](https://github.com/rancher/fleet/blob/9e035bc485316114a91b0f865f9e899e5c5b02a2/package/Dockerfile.agent#L4), so they would expect the binary to behave like busybox variants instead of GNU ones